sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2819,6 +2911,12 @@ public partial class GremlinDatabaseResourceInfo
         public GremlinDatabaseResourceInfo(string databaseName) { }
         public string DatabaseName { get { throw null; } set { } }
     }
+    public partial class GremlinDatabaseRestoreResource
```

Comment:
Shall we rename this to `GremlinDatabaseRestoreInfo`? As I said in other comments, `Resource` is a reserved suffix in dotnet mgmt SDK

Reply:
We already have this change in preview version and we are just making it stable. Also, we are following similar convention for existing APIs of sql and mongo so we decided to use same for these new ones as well to remain consistent.